node/src/components/storage/tests.rs

```diff
@@ -907,7 +907,7 @@ fn should_hard_reset() {
             let is_switch = index % blocks_per_era == blocks_per_era - 1;
             Block::random_with_specifics(
                 &mut harness.rng,
-                EraId(index as u64 / 3),
+                EraId::from(index as u64 / 3),
```

Comment:
Looks like `blocks_count` should be a `u64` and `index` should be called `height`.